tests/pseudopotential/vloc_nonrel_dft.f90

```diff
@@ -104,10 +104,10 @@ program vloc_nonrel_dft
 print *, V_h(:10)
 
 ! Save the radial grid, density, V_h
-open(newunit(u), file="density.txt", status="replace")
-write(u, "((es23.16, ' ', es23.16, ' ', es23.16))") (R(i), density(i), V_h(i), &
-        i=1, size(R))
-close(u)
+!open(newunit(u), file="density.txt", status="replace")
+!write(u, "((es23.16, ' ', es23.16, ' ', es23.16))") (R(i), density(i), V_h(i), &
+!        i=1, size(R))
+!close(u)
```

Comment:
Fixed in main, but prints to stdout: https://github.com/lfortran/lfortran/issues/2720